cityofaustin/atd-moped
# Dropdown order of project roles, status, and phases should match in Gitbook #5857

Closes https://github.com/cityofaustin/atd-data-tech/issues/5857

To test:

```
1. Run $ ./hasura-cluster restart
2. Run $ npm run start
```

Check the status order, the roles order, and phase order.
